### ISSUE TITLE
Restore alphbetized package listing order when installing (or at least allow this via config option)

### ISSUE THREAD

**Issue by DManowitz** [mamba-org/mamba#3273]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Anaconda default channels

- [X] I do NOT use the Anaconda default channels (pkgs/* etc.)

### How did you install Mamba?

Other (please describe)

### Search tried in issue tracker

install package order

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

I have this problem with Conda as well, without using Mamba

### Describe your issue

When installing packages using mamba v1.4.7 or earlier, the packages to be installed are listed in ascending alphabetical order.  However, starting with mamba v1.4.9, the packages to be installed are listed in an arbitrary order.  

Having the packages listed in alphabetical order makes it much easier to verify if certain dependencies are being installed correctly, so I would like the previous behavior restored.  If there is some compelling reason to not make alphabetizing the default behavior, then please at least make it a configuration option (and don't forget to add that option to the documentation in that case!)

### mamba info / micromamba info

```shell
N/A
```

### Logs

```shell
N/A
```

### environment.yml

```yaml
N/A
```

### ~/.condarc

```yaml
N/A
```

**Comment by Hind-M** [comment]
Can you please add an example showing this problem?

**Comment by DManowitz** [comment]
If I try to install geoviews using mamba 1.4.7, I get the following output:

```
(py38_test) C:\Users\manow>mamba install geoviews

Looking for: ['geoviews']

conda-forge/win-64                                          Using cache
conda-forge/noarch                                          Using cache

Pinned packages:
  - python 3.8.*

Transaction

  Prefix: C:\Users\manow\miniconda3\envs\py38_test

  Updating specs:

   - geoviews
   - ca-certificates
   - openssl


  Package                               Version  Build                   Channel           Size
-------------------------------------------------------------------------------------------------
  Install:
-------------------------------------------------------------------------------------------------

  + bleach                                6.1.0  pyhd8ed1ab_0            conda-forge     Cached
  + blosc                                1.21.5  hbd69f2e_1              conda-forge       50kB
  + bokeh                                 3.1.1  pyhd8ed1ab_0            conda-forge        6MB
  + brotli                                1.1.0  hcfcfb64_1              conda-forge     Cached
  + brotli-bin                            1.1.0  hcfcfb64_1              conda-forge     Cached
  + brotli-python                         1.1.0  py38hd3f51b4_1          conda-forge      322kB
  + cartopy                              0.21.1  py38hf3113ef_1          conda-forge        2MB
  + certifi                            2024.2.2  pyhd8ed1ab_0            conda-forge     Cached
  + cftime                                1.6.3  py38he7056a7_0          conda-forge      186kB
  + charset-normalizer                    3.3.2  pyhd8ed1ab_0            conda-forge     Cached
  + click                                 8.1.7  win_pyh7428d3b_0        conda-forge       85kB
  + cloudpickle                           3.0.0  pyhd8ed1ab_0            conda-forge       25kB
  + colorama                              0.4.6  pyhd8ed1ab_0            conda-forge     Cached
  + colorcet                              3.1.0  pyhd8ed1ab_0            conda-forge     Cached
  + contourpy                             1.1.1  py38hb1fd069_1          conda-forge      174kB
  + cycler                               0.12.1  pyhd8ed1ab_0            conda-forge     Cached
  + dask-core                          2023.5.0  pyhd8ed1ab_0            conda-forge      845kB
  + datashader                           0.15.2  pyhd8ed1ab_0            conda-forge       17MB
  + datashape                             0.5.4  py_1                    conda-forge       50kB
  + fonttools                            4.51.0  py38h91455d4_0          conda-forge        2MB
  + freetype                             2.12.1  hdaf720e_2              conda-forge     Cached
  + fsspec                             2024.3.1  pyhca7485f_0            conda-forge      129kB
  + geopandas-base                       0.13.2  pyha770c72_1            conda-forge        1MB
  + geos                                 3.11.2  h1537add_0              conda-forge        1MB
  + geoviews                             1.10.1  pyhd8ed1ab_0            conda-forge        9kB
  + geoviews-core                        1.10.1  pyha770c72_0            conda-forge      373kB
  + hdf4                                 4.2.15  h1334946_6              conda-forge      780kB
  + hdf5                                 1.14.3  nompi_h73e8ff5_100      conda-forge        2MB
  + holoviews                            1.17.1  pyhd8ed1ab_0            conda-forge        4MB
  + idna                                    3.7  pyhd8ed1ab_0            conda-forge     Cached
  + importlib-metadata                    7.1.0  pyha770c72_0            conda-forge     Cached
  + importlib-resources                   6.4.0  pyhd8ed1ab_0            conda-forge       10kB
  + importlib_metadata                    7.1.0  hd8ed1ab_0              conda-forge     Cached
  + importlib_resources                   6.4.0  pyhd8ed1ab_0            conda-forge     Cached
  + intel-openmp                       2024.1.0  h57928b3_965            conda-forge        2MB
  + jinja2                                3.1.3  pyhd8ed1ab_0            conda-forge     Cached
  + kiwisolver                            1.4.5  py38hb1fd069_1          conda-forge       56kB
  + krb5                                 1.21.2  heb0366b_0              conda-forge     Cached
  + lcms2                                  2.15  h3e3b177_1              conda-forge      499kB
  + lerc                                  4.0.0  h63175ca_0              conda-forge      194kB
  + libaec                                1.1.3  h63175ca_0              conda-forge       33kB
  + libblas                               3.9.0  22_win64_mkl            conda-forge        5MB
  + libbrotlicommon                       1.1.0  hcfcfb64_1              conda-forge     Cached
  + libbrotlidec                          1.1.0  hcfcfb64_1              conda-forge     Cached
  + libbrotlienc                          1.1.0  hcfcfb64_1              conda-forge     Cached
  + libcblas                              3.9.0  22_win64_mkl            conda-forge        5MB
  + libcurl                               8.7.1  hd5e4a3a_0              conda-forge     Cached
  + libdeflate                             1.18  hcfcfb64_0              conda-forge      152kB
  + libhwloc                             2.10.0  default_h2fffb23_1000   conda-forge        2MB
  + libiconv                               1.17  hcfcfb64_2              conda-forge     Cached
  + libjpeg-turbo                       2.1.5.1  hcfcfb64_1              conda-forge      688kB
  + liblapack                             3.9.0  22_win64_mkl            conda-forge        5MB
  + libnetcdf                             4.9.2  nompi_h07c049d_113      conda-forge      625kB
  + libpng                               1.6.43  h19919ed_0              conda-forge     Cached
  + libssh2                              1.11.0  h7dfc565_0              conda-forge     Cached
  + libtiff                               4.5.1  h6c8260b_1              conda-forge      953kB
  + libwebp-base                          1.4.0  hcfcfb64_0              conda-forge      274kB
  + libxcb                                 1.15  hcd874cb_0              conda-forge      970kB
  + libxml2                              2.12.6  hc3477c8_2              conda-forge        2MB
  + libzip                               1.10.1  h1d365fa_3              conda-forge      146kB
  + linkify-it-py                         2.0.3  pyhd8ed1ab_0            conda-forge       24kB
  + llvmlite                             0.41.1  py38h19421c1_0          conda-forge       17MB
  + locket                                1.0.0  pyhd8ed1ab_0            conda-forge        8kB
  + lz4-c                                 1.9.4  hcfcfb64_0              conda-forge      134kB
  + m2w64-gcc-libgfortran                 5.3.0  6                       conda-forge      351kB
  + m2w64-gcc-libs                        5.3.0  7                       conda-forge      532kB
  + m2w64-gcc-libs-core                   5.3.0  7                       conda-forge      219kB
  + m2w64-gmp                             6.1.0  2                       conda-forge      744kB
  + m2w64-libwinpthread-git  5.0.0.4634.697f757  2                       conda-forge       32kB
  + markdown                                3.6  pyhd8ed1ab_0            conda-forge       78kB
  + markdown-it-py                        3.0.0  pyhd8ed1ab_0            conda-forge       64kB
  + markupsafe                            2.1.5  py38h91455d4_0          conda-forge       27kB
  + matplotlib-base                       3.7.3  py38h2724991_0          conda-forge        7MB
  + mdit-py-plugins                       0.4.0  pyhd8ed1ab_0            conda-forge       41kB
  + mdurl                                 0.1.2  pyhd8ed1ab_0            conda-forge       15kB
  + mkl                                2024.1.0  h66d3029_692            conda-forge      109MB
  + msys2-conda-epoch                  20160418  1                       conda-forge        3kB
  + multipledispatch                      0.6.0  py_0                    conda-forge       12kB
  + munkres                               1.1.4  pyh9f0ad1d_0            conda-forge       12kB
  + netcdf4                               1.6.5  nompi_py38h46ce47e_100  conda-forge      414kB
  + numba                                0.58.1  py38h4a59444_0          conda-forge        4MB
  + numpy                                1.24.4  py38h1d91fd2_0          conda-forge        6MB
  + openjpeg                              2.5.0  ha2aaf27_2              conda-forge     Cached
  + packaging                              24.0  pyhd8ed1ab_0            conda-forge       50kB
  + pandas                                1.5.3  py38h5846ac1_1          conda-forge       11MB
  + panel                                 1.2.3  pyhd8ed1ab_0            conda-forge       14MB
  + param                                1.13.0  pyh1a96a4e_0            conda-forge       80kB
  + partd                                 1.4.1  pyhd8ed1ab_0            conda-forge       21kB
  + pillow                               10.0.0  py38ha7eb54a_0          conda-forge       47MB
  + platformdirs                          4.2.1  pyhd8ed1ab_0            conda-forge       20kB
  + pooch                                 1.8.1  pyhd8ed1ab_0            conda-forge     Cached
  + proj                                  9.2.0  heca977f_0              conda-forge        3MB
  + pthread-stubs                           0.4  hcd874cb_1001           conda-forge        6kB
  + pthreads-win32                        2.9.1  hfa6e2cd_3              conda-forge      144kB
  + pyct                                  0.5.0  pyhd8ed1ab_0            conda-forge       20kB
  + pyparsing                             3.1.2  pyhd8ed1ab_0            conda-forge       89kB
  + pyproj                                3.5.0  py38h24c0b3c_1          conda-forge      387kB
  + pyshp                                 2.3.1  pyhd8ed1ab_0            conda-forge      964kB
  + pysocks                               1.7.1  py38haa244fe_5          conda-forge     Cached
  + python-dateutil                       2.9.0  pyhd8ed1ab_0            conda-forge     Cached
  + python_abi                              3.8  4_cp38                  conda-forge        7kB
  + pytz                                 2024.1  pyhd8ed1ab_0            conda-forge     Cached
  + pyviz_comms                           3.0.1  pyhd8ed1ab_0            conda-forge       48kB
  + pyyaml                                6.0.1  py38h91455d4_1          conda-forge      152kB
  + requests                             2.31.0  pyhd8ed1ab_0            conda-forge     Cached
  + scipy                                1.10.1  py38h1aea9ed_3          conda-forge       18MB
  + shapely                               2.0.1  py38h49d2d47_1          conda-forge      392kB
  + six                                  1.16.0  pyh6c4a22f_0            conda-forge     Cached
  + snappy                                1.2.0  hfb803bf_1              conda-forge       60kB
  + sqlite                               3.45.3  hcfcfb64_0              conda-forge     Cached
  + tbb                               2021.12.0  h91493d7_0              conda-forge      162kB
  + toolz                                0.12.1  pyhd8ed1ab_0            conda-forge     Cached
  + tornado                                 6.4  py38h91455d4_0          conda-forge      641kB
  + tqdm                                 4.66.2  pyhd8ed1ab_0            conda-forge     Cached
  + typing_extensions                    4.11.0  pyha770c72_0            conda-forge     Cached
  + uc-micro-py                           1.0.3  pyhd8ed1ab_0            conda-forge       11kB
  + unicodedata2                         15.1.0  py38h91455d4_0          conda-forge      371kB
  + urllib3                               2.2.1  pyhd8ed1ab_0            conda-forge     Cached
  + webencodings                          0.5.1  pyhd8ed1ab_2            conda-forge     Cached
  + win_inet_pton                         1.1.0  py38haa244fe_5          conda-forge     Cached
  + xarray                             2023.1.0  pyhd8ed1ab_0            conda-forge      663kB
  + xorg-libxau                          1.0.11  hcd874cb_0              conda-forge       51kB
  + xorg-libxdmcp                         1.1.3  hcd874cb_0              conda-forge       68kB
  + xyzservices                        2024.4.0  pyhd8ed1ab_0            conda-forge       46kB
  + yaml                                  0.2.5  h8ffe710_2              conda-forge     Cached
  + zipp                                 3.17.0  pyhd8ed1ab_0            conda-forge     Cached
  + zlib                                 1.2.13  hcfcfb64_5              conda-forge     Cached
  + zstd                                  1.5.5  h12be248_0              conda-forge     Cached

  Reinstall:
-------------------------------------------------------------------------------------------------

  o openssl                               3.2.1  hcfcfb64_1              conda-forge     Cached

  Summary:

  Install: 128 packages
  Reinstall: 1 packages

  Total download: 306MB

-------------------------------------------------------------------------------------------------
```

However, if I try to install geoviews using mamba 1.5.8, I get the following output:

```
(py38_test) C:\Users\manow>mamba install geoviews

Looking for: ['geoviews']

conda-forge/noarch                                            No change
conda-forge/win-64                                            No change

Pinned packages:
  - python 3.8.*

Transaction

  Prefix: C:\Users\manow\miniconda3\envs\py38_test

  Updating specs:

   - geoviews
   - ca-certificates
   - openssl


  Package                               Version  Build                   Channel           Size
-------------------------------------------------------------------------------------------------
  Install:
-------------------------------------------------------------------------------------------------

  + geos                                 3.11.2  h1537add_0              conda-forge        1MB
  + python_abi                              3.8  4_cp38                  conda-forge        7kB
  + intel-openmp                       2024.1.0  h57928b3_965            conda-forge        2MB
  + pthreads-win32                        2.9.1  hfa6e2cd_3              conda-forge      144kB
  + msys2-conda-epoch                  20160418  1                       conda-forge        3kB
  + yaml                                  0.2.5  h8ffe710_2              conda-forge     Cached
  + sqlite                               3.45.3  hcfcfb64_0              conda-forge     Cached
  + lerc                                  4.0.0  h63175ca_0              conda-forge      194kB
  + libdeflate                             1.18  hcfcfb64_0              conda-forge      152kB
  + libjpeg-turbo                       2.1.5.1  hcfcfb64_1              conda-forge      688kB
  + libwebp-base                          1.4.0  hcfcfb64_0              conda-forge      274kB
  + libpng                               1.6.43  h19919ed_0              conda-forge     Cached
  + zlib                                 1.2.13  hcfcfb64_5              conda-forge     Cached
  + zstd                                  1.5.5  h12be248_0              conda-forge     Cached
  + libiconv                               1.17  hcfcfb64_2              conda-forge     Cached
  + snappy                                1.2.0  hfb803bf_1              conda-forge       60kB
  + libaec                                1.1.3  h63175ca_0              conda-forge       33kB
  + libbrotlicommon                       1.1.0  hcfcfb64_1              conda-forge     Cached
  + lz4-c                                 1.9.4  hcfcfb64_0              conda-forge      134kB
  + libzip                               1.10.1  h1d365fa_3              conda-forge      146kB
  + libssh2                              1.11.0  h7dfc565_0              conda-forge     Cached
  + krb5                                 1.21.2  heb0366b_0              conda-forge     Cached
  + win_inet_pton                         1.1.0  py38haa244fe_5          conda-forge     Cached
  + brotli-python                         1.1.0  py38hd3f51b4_1          conda-forge      322kB
  + markupsafe                            2.1.5  py38h91455d4_0          conda-forge       27kB
  + unicodedata2                         15.1.0  py38h91455d4_0          conda-forge      371kB
  + kiwisolver                            1.4.5  py38hb1fd069_1          conda-forge       56kB
  + tornado                                 6.4  py38h91455d4_0          conda-forge      641kB
  + llvmlite                             0.41.1  py38h19421c1_0          conda-forge       17MB
  + m2w64-libwinpthread-git  5.0.0.4634.697f757  2                       conda-forge       32kB
  + m2w64-gmp                             6.1.0  2                       conda-forge      744kB
  + pyyaml                                6.0.1  py38h91455d4_1          conda-forge      152kB
  + hdf4                                 4.2.15  h1334946_6              conda-forge      780kB
  + freetype                             2.12.1  hdaf720e_2              conda-forge     Cached
  + libtiff                               4.5.1  h6c8260b_1              conda-forge      953kB
  + libxml2                              2.12.6  hc3477c8_2              conda-forge        2MB
  + libbrotlienc                          1.1.0  hcfcfb64_1              conda-forge     Cached
  + libbrotlidec                          1.1.0  hcfcfb64_1              conda-forge     Cached
  + blosc                                1.21.5  hbd69f2e_1              conda-forge       50kB
  + libcurl                               8.7.1  hd5e4a3a_0              conda-forge     Cached
  + pysocks                               1.7.1  py38haa244fe_5          conda-forge     Cached
  + m2w64-gcc-libs-core                   5.3.0  7                       conda-forge      219kB
  + lcms2                                  2.15  h3e3b177_1              conda-forge      499kB
  + openjpeg                              2.5.0  ha2aaf27_2              conda-forge     Cached
  + libhwloc                             2.10.0  default_h2fffb23_1000   conda-forge        2MB
  + brotli-bin                            1.1.0  hcfcfb64_1              conda-forge     Cached
  + hdf5                                 1.14.3  nompi_h73e8ff5_100      conda-forge        2MB
  + proj                                  9.2.0  heca977f_0              conda-forge        3MB
  + m2w64-gcc-libgfortran                 5.3.0  6                       conda-forge      351kB
  + tbb                               2021.12.0  h91493d7_0              conda-forge      162kB
  + brotli                                1.1.0  hcfcfb64_1              conda-forge     Cached
  + libnetcdf                             4.9.2  nompi_h07c049d_113      conda-forge      625kB
  + m2w64-gcc-libs                        5.3.0  7                       conda-forge      532kB
  + mkl                                2024.1.0  h66d3029_692            conda-forge      109MB
  + xorg-libxdmcp                         1.1.3  hcd874cb_0              conda-forge       68kB
  + pthread-stubs                           0.4  hcd874cb_1001           conda-forge        6kB
  + xorg-libxau                          1.0.11  hcd874cb_0              conda-forge       51kB
  + libblas                               3.9.0  22_win64_mkl            conda-forge        5MB
  + libxcb                                 1.15  hcd874cb_0              conda-forge      970kB
  + libcblas                              3.9.0  22_win64_mkl            conda-forge        5MB
  + liblapack                             3.9.0  22_win64_mkl            conda-forge        5MB
  + pillow                               10.0.0  py38ha7eb54a_0          conda-forge       47MB
  + numpy                                1.24.4  py38h1d91fd2_0          conda-forge        6MB
  + cftime                                1.6.3  py38he7056a7_0          conda-forge      186kB
  + contourpy                             1.1.1  py38hb1fd069_1          conda-forge      174kB
  + shapely                               2.0.1  py38h49d2d47_1          conda-forge      392kB
  + packaging                              24.0  pyhd8ed1ab_0            conda-forge       50kB
  + pyshp                                 2.3.1  pyhd8ed1ab_0            conda-forge      964kB
  + param                                1.13.0  pyh1a96a4e_0            conda-forge       80kB
  + toolz                                0.12.1  pyhd8ed1ab_0            conda-forge     Cached
  + colorcet                              3.1.0  pyhd8ed1ab_0            conda-forge     Cached
  + cloudpickle                           3.0.0  pyhd8ed1ab_0            conda-forge       25kB
  + fsspec                             2024.3.1  pyhca7485f_0            conda-forge      129kB
  + colorama                              0.4.6  pyhd8ed1ab_0            conda-forge     Cached
  + xyzservices                        2024.4.0  pyhd8ed1ab_0            conda-forge       46kB
  + six                                  1.16.0  pyh6c4a22f_0            conda-forge     Cached
  + certifi                            2024.2.2  pyhd8ed1ab_0            conda-forge     Cached
  + pytz                                 2024.1  pyhd8ed1ab_0            conda-forge     Cached
  + cycler                               0.12.1  pyhd8ed1ab_0            conda-forge     Cached
  + pyparsing                             3.1.2  pyhd8ed1ab_0            conda-forge       89kB
  + munkres                               1.1.4  pyh9f0ad1d_0            conda-forge       12kB
  + idna                                    3.7  pyhd8ed1ab_0            conda-forge     Cached
  + charset-normalizer                    3.3.2  pyhd8ed1ab_0            conda-forge     Cached
  + platformdirs                          4.2.1  pyhd8ed1ab_0            conda-forge       20kB
  + locket                                1.0.0  pyhd8ed1ab_0            conda-forge        8kB
  + typing_extensions                    4.11.0  pyha770c72_0            conda-forge     Cached
  + mdurl                                 0.1.2  pyhd8ed1ab_0            conda-forge       15kB
  + uc-micro-py                           1.0.3  pyhd8ed1ab_0            conda-forge       11kB
  + zipp                                 3.17.0  pyhd8ed1ab_0            conda-forge     Cached
  + webencodings                          0.5.1  pyhd8ed1ab_2            conda-forge     Cached
  + jinja2                                3.1.3  pyhd8ed1ab_0            conda-forge     Cached
  + urllib3                               2.2.1  pyhd8ed1ab_0            conda-forge     Cached
  + pyviz_comms                           3.0.1  pyhd8ed1ab_0            conda-forge       48kB
  + tqdm                                 4.66.2  pyhd8ed1ab_0            conda-forge     Cached
  + click                                 8.1.7  win_pyh7428d3b_0        conda-forge       85kB
  + python-dateutil                       2.9.0  pyhd8ed1ab_0            conda-forge     Cached
  + multipledispatch                      0.6.0  py_0                    conda-forge       12kB
  + partd                                 1.4.1  pyhd8ed1ab_0            conda-forge       21kB
  + markdown-it-py                        3.0.0  pyhd8ed1ab_0            conda-forge       64kB
  + linkify-it-py                         2.0.3  pyhd8ed1ab_0            conda-forge       24kB
  + importlib_resources                   6.4.0  pyhd8ed1ab_0            conda-forge     Cached
  + importlib-metadata                    7.1.0  pyha770c72_0            conda-forge     Cached
  + bleach                                6.1.0  pyhd8ed1ab_0            conda-forge     Cached
  + requests                             2.31.0  pyhd8ed1ab_0            conda-forge     Cached
  + datashape                             0.5.4  py_1                    conda-forge       50kB
  + mdit-py-plugins                       0.4.0  pyhd8ed1ab_0            conda-forge       41kB
  + importlib-resources                   6.4.0  pyhd8ed1ab_0            conda-forge       10kB
  + markdown                                3.6  pyhd8ed1ab_0            conda-forge       78kB
  + importlib_metadata                    7.1.0  hd8ed1ab_0              conda-forge     Cached
  + pooch                                 1.8.1  pyhd8ed1ab_0            conda-forge     Cached
  + pyct                                  0.5.0  pyhd8ed1ab_0            conda-forge       20kB
  + dask-core                          2023.5.0  pyhd8ed1ab_0            conda-forge      845kB
  + netcdf4                               1.6.5  nompi_py38h46ce47e_100  conda-forge      414kB
  + pyproj                                3.5.0  py38h24c0b3c_1          conda-forge      387kB
  + fonttools                            4.51.0  py38h91455d4_0          conda-forge        2MB
  + pandas                                1.5.3  py38h5846ac1_1          conda-forge       11MB
  + numba                                0.58.1  py38h4a59444_0          conda-forge        4MB
  + scipy                                1.10.1  py38h1aea9ed_3          conda-forge       18MB
  + matplotlib-base                       3.7.3  py38h2724991_0          conda-forge        7MB
  + cartopy                              0.21.1  py38hf3113ef_1          conda-forge        2MB
  + geopandas-base                       0.13.2  pyha770c72_1            conda-forge        1MB
  + xarray                             2023.1.0  pyhd8ed1ab_0            conda-forge      663kB
  + bokeh                                 3.1.1  pyhd8ed1ab_0            conda-forge        6MB
  + datashader                           0.15.2  pyhd8ed1ab_0            conda-forge       17MB
  + panel                                 1.2.3  pyhd8ed1ab_0            conda-forge       14MB
  + holoviews                            1.17.1  pyhd8ed1ab_0            conda-forge        4MB
  + geoviews-core                        1.10.1  pyha770c72_0            conda-forge      373kB
  + geoviews                             1.10.1  pyhd8ed1ab_0            conda-forge        9kB

  Reinstall:
-------------------------------------------------------------------------------------------------

  o openssl                               3.2.1  hcfcfb64_1              conda-forge     Cached

  Summary:

  Install: 128 packages
  Reinstall: 1 packages

  Total download: 306MB

-------------------------------------------------------------------------------------------------
```

I would like (either by default, by a command line option, and/or a config file option) to have the listing of packages to be installed be alphabetized, as it has been with mamba versions <= 1.4.7.

**Comment by DManowitz** [comment]
Closed via #3146
